src/h5web/storage-utils.ts

```diff
@@ -0,0 +1,54 @@
+import create, { State, StateCreator, StoreApi, UseStore } from 'zustand';
+
+export interface StorageConfig {
+  key: string;
+  whitelist: string[];
```

Comment:
I get why you called it `whitelist` but I find the name could be more meaningful.

`itemsToPersist`?, `persistentItems`?, `itemsToStore` ?